frontend_tests/casper_lib/common.js

```diff
@@ -56,7 +56,7 @@ exports.initialize_casper = function (viewport) {
     casper.test.on('fail', function failure() {
         if (casper_failure_count <= 10) {
             casper.capture("var/casper/casper-failure" + casper_failure_count + ".png");
-            casper_failure_count++;
+            casper_failure_count+=1;
```

Comment:
One would usually write this with more whitespace, e.g. `casper_failure_count += 1`.